### PR TITLE
Migrate base-ui screenshot tests

### PR DESCRIPTION
**Pull request by yschimke**
#### WHAT

Migrate base-ui screenshot tests

#### WHY

Remove paparazzi

#### Checklist :clipboard:
- [x] Add explicit visibility modifier and explicit return types for public declarations
- [x] Run spotless check
- [x] Run tests
- [x] Update metalava's signature text files

### CODE REVIEW

File: base-ui/src/test/snapshots/images/com.google.android.horologist.base.ui.components_PrimaryChipTest_disabled.png
Comment (luizgrp) [inline base-ui/src/test/snapshots/images/com.google.android.horologist.base.ui.components_PrimaryChipTest_disabled.png:1]: it looks like there is more transparency applied to the chip background

**Review by luizgrp — APPROVED** [review]

File: base-ui/src/test/snapshots/images/com.google.android.horologist.base.ui.components_PrimaryChipTest_disabled.png
Comment (yschimke) [reply base-ui/src/test/snapshots/images/com.google.android.horologist.base.ui.components_PrimaryChipTest_disabled.png:1]: Agreed with Filip on Friday, but this one surprised me.  We wanted the transparency to be visible, but not distracting, so went with black, 50% alpha.  Are we expecting these to have transparent chip background?

Or do you think it's an RNG bug?

cc @fstanis

Comment (luizgrp) [reply base-ui/src/test/snapshots/images/com.google.android.horologist.base.ui.components_PrimaryChipTest_disabled.png:1]: it looks like wear compose material applies alpha to the contents of the chip when in disabled state

will check it with some sample code at some point

Comment (yschimke) [reply base-ui/src/test/snapshots/images/com.google.android.horologist.base.ui.components_PrimaryChipTest_disabled.png:1]: So this caught something! nice.

**Pull request merged**